### ISSUE TITLE
{Linked:Bug3750479} [Designer] [A11y] There should be single pointer to drag control's code to editor area.

### ISSUE THREAD

**Issue by osheengithub** [microsoft/AdaptiveCards#8898]
### Problem Description

User experience: 
Users who cannot use mouse/keyboard to drag, there should be single pointer to move code blocks.

Test Environment:
Platform: Web
OS: Windows_11 Version: 22H2
OS Build: 22598.200
Browser: Chrome 123.0.6312.107 (Official Build) (64-bit)
Ring Version: Canary (R0)

Pre-requisites:
 Login with valid credentials.
Repro steps:

There should be sigle pointer to drag control's code to editor area.
Actual Result:
There is no sigle pointer to drag control's code to editor area.
Expected Result:
There should be sigle pointer to drag control's code to editor area.

![image](https://github.com/microsoft/AdaptiveCards/assets/47814799/b198bbaf-3288-45cc-af22-9bb9755e8066)

WCAG Reference: [Understanding Success Criterion 2.5.7: Dragging Movements | WAI | W3C](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.6",
    "body": []
}
```

**Comment by anna-dingler** [comment]
Hi @osheengithub,

Can you describe what you mean by "single pointer"?

Thanks,
Anna

**Comment by anna-dingler** [comment]
Synced with the accessibility team offline, and this bug can be closed.

Elements can be added to the canvas with two methods in addition to drag and drop:
1. Double clicking on the element
2. Pressing enter when the element has focus

This meets accessibility guidelines.
